R/paste.R

```diff
@@ -0,0 +1,24 @@
+#' "Paste" together factors.
+#'
+#' Order is cartesian product of all levels.
+#'
+#' This is sometimes useful when plotting a factor
```

Comment:
Good call. I would recommend implementing it using `base::interaction()` if this feature is worthwhile.